### ISSUE TITLE
unlock folder results in 403

### ISSUE THREAD

**Issue by oldenboom** [danmunn/redmine_dmsf#1258]
When unlocking a folder the project-number is used in the path instead of the path-name of the project. For instance:

`<a title="Locked by me" class="icon icon-unlock" href="/projects/1/dmsf/unlock?folder_id=1">Unlock</a>`

When I replace the URL with `/projects/project-path/dmsf/unlock?folder_id=1` unlocking does function properly.

**Comment by picman** [comment]
Fixed.

**Comment by serberman** [comment]
2.4.8 devel
Redmine don't start after fix #1258

```
Creating scope :system. Overwriting existing method Enumeration.system.
DEPRECATION WARNING: Using a dynamic :action segment in a route is deprecated and will be removed in Rails 6.0. (called from instance_eval at C:/Bitnami/redmine-4.2.1-0/apps/redmine/htdocs/config/routes.rb:395)
```

**Comment by picman** [comment]
You should provide more information. The "DEPRECATION WARNING" is for sure not the cause.
